### DISCUSSION TITLE
Any plans to support Angular 17 Control Flow Migration

### DISCUSSION THREAD

**Discussion by shresthaujjwal** [openrewrite/rewrite#3786] [Ideas] [answered]
With Angular 17 new control flow is being introduced. This is a major change and requires lot of refactoring across company. This is major saving to everyone. OR does support xml which is probably help a little with control flow changes as html is based on xml. But it still needs more support and needs to be streamline and tested. Thoughts plans ?
https://angular.io/guide/control_flow

**Comment by timtebeek** [comment, marked as answer]
Hi @shresthaujjwal ! Looks like something we'd want to cover in the future, as part of [our plans to expand into Typescript](https://docs.moderne.io/administrator-documentation/references/language-roadmap) and the ecosystem that comes with it. Be sure to keep an eye on new releases, especially going into the new year!

**Reply by shresthaujjwal** [reply]
Awesome thank you
